interface-impl-v2/src/main/java/org/mule/apikit/implv2/ParserV2Utils.java

```diff
@@ -147,6 +148,9 @@ private static void findIncludeNodes(String rootPath, final String pathRelativeT
   }
 
   private static String getParent(URI uri) {
+    if(ApiSyncUtils.isSyncProtocol(uri.toString())){
```

Comment:
If it is a ApiSync resource yo don't need to get the parent, the ApiSyncResourceLoader handles that.